### PR TITLE
Add Helm chart for Infection Monkey deployment

### PR DESCRIPTION
**Pull request by MikeNikolayev**
# What does this PR do?

This PR introduces a Helm chart for deploying Infection Monkey resources on a Kubernetes cluster. The chart simplifies the deployment process by defining Kubernetes resources such as:

ServiceAccount
ClusterRoleBinding
Deployment (with Monkey container and MongoDB)
Service (NodePort for accessing the Infection Monkey UI)

## PR Checklist
* [* ] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running helm lint to verify the Helm chart syntax and structure}
    > Tested by {Running helm install/unintall}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

**Review by mssalvatore — COMMENTED** [review]
Thanks for the contribution!

This is a nice idea, but, unfortunately, I don't think we should merge it. The project is a bit understaffed at the moment and this is another thing we'd have to maintain during development and test at release time. My fear is that after it gets merged, it will get ignored and fall into a state of disrepair.

In addition, without any documentation it's unlikely for anyone to use this. If no one will use it (because they don't know about it), then it doesn't provide any value. We could add documentation to the PR, but the same arguments for maintaining the helm chart also apply to maintaining its documentation.

In summary, I sincerely appreciate the contribution, but I'm reluctant to take on the additional maintenance burden at this time.
